code/modules/research/xenoarchaeology/finds/finds_guitar.dm

```diff
@@ -0,0 +1,28 @@
+/obj/item/device/instrument/guitar/magical
+	desc = "An ancient guitar, from the legendary artist born on Mars. It was said that he was a son of a god, gifted with the power of lighting himself. People say that he was the most humble man.. and that his music was simply.. electric!"
+	force = 15
+	mech_flags = MECH_SCAN_FAIL
+
+/obj/item/device/instrument/guitar/magical/afterattack(atom/target, mob/user, proximity_flag)
+	..()
+	if(prob(20) && (wielded && song.playing) && proximity_flag) //20% chance to fire 4 bolts of lighting(33% damage each).
+		for(var/i=0, i<4, i++)
+			eletric_music(target, user)
+
+/obj/item/device/instrument/guitar/magical/proc/eletric_music(var/atom/target, var/mob/user)
+	var/turf/T = get_turf(user)
+	var/turf/U = get_turf(target)
+	var/obj/item/projectile/beam/lightning/L = getFromPool(/obj/item/projectile/beam/lightning, T)
```

Comment:
could use the generic_fire proc? generic_projectile_fire()?

Reply:
I tried but then i'd be unable to mess with damage of the projectile